src/test/java/org/reaktivity/nukleus/tcp/internal/streams/rfc793/ClientIOExceptionFromWriteIT.java

```diff
@@ -97,6 +98,35 @@ public void shouldResetWhenImmediateWriteThrowsIOException() throws Exception
     @Test
     @Specification({
         "${route}/client/controller",
+        "${client}/client.sent.data.received.abort.and.reset/client"
+    })
+    @BMRule(name = "processData",
+    targetClass = "^java.nio.channels.SocketChannel",
+    targetMethod = "write(java.nio.ByteBuffer)",
+    condition =
+      "callerEquals(\"org.reaktivity.nukleus.tcp.internal.stream.WriteStream.processData\", true, true)",
+      action = "throw new IOException(\"Simulating an IOException from write\")"
+    )
+    public void shouldAbortWhenImmediateWriteThrowsIOException() throws Exception
```

Comment:
This test is a duplicate of the following one (client.send.data.received.reset.and.abort) because the order in which the client receives reset and abort is unpredictable and is not significant since they are read from different places: abort is read from the streams file on the stream coming into the client, reset is read from the throttle on the stream going out from the client (the stream it writes to). What we are testing here is that WriteStream.handleIOExceptionFromWrite correctly writes an Abort frame to the client's input stream (where they are reading from, so causing read aborted) and writes a Reset frame the client output stream throttle (causing write aborted).  client.sent.data.received.reset.and.abort is already sufficient verification of that.

Reply:
I've removed it.